### DISCUSSION TITLE
Bootstrap CSS's Popover & Tooltip are not working

### DISCUSSION THREAD

**Discussion by naveencom** [unpoly/unpoly#526] [Q&A]
- https://getbootstrap.com/docs/5.3/components/popovers/
- https://getbootstrap.com/docs/5.3/components/tooltips/

No errors in the console area.

Any help ?

**Comment by naveencom** [comment]
@adam12 @triskweline Please give me some hint at least.

**Reply by adam12** [reply]
Do you feel this is an Unpoly issue? If so, why? You didn't provide any example code or a reproduction, so there's not much to go on.

Just as an aside, Unpoly is a passion project for both of us, so tagging us 2 days after asking a question to poke us for information to your very anemic question is very disrespectful. At least 1 of those days was a public holiday for me before the kids are back to school.

**Reply by naveencom** [reply]
Adam, Please forgive me. 

I didn't mean to be disrespectful.

Thanks for your time.

**Comment by vb8448** [comment]
Hi, i guess the issue is because bootstrap doesn't see new elements added by unpoly. 

You can run a compiler function like `$('[data-bs-toggle=\"tooltip\"]').tooltip(); ` to fix it.
